test/service/test_dss_proxy.py

```diff
@@ -81,6 +81,7 @@ def test_dss_files_proxy(self):
                 'version': file_version
             }).add(path=file_uuid)
         dss_token = 'some_token'
+        request_to_index = '1'
```

Comment:
Inline. Used only once, doesn't add clarity. No need to extract variable. Also what's up with the "to"? I don't get why these weird inconsistencies pop up occasionally. It's almost like there's intent to trip the reader up.